src/users/interfaces/authenticate-managed-user-options.interface.ts

```diff
@@ -0,0 +1,8 @@
+export interface AuthenticateManagedUserOptions {
+  client_id: string;
+  client_secret: string;
+  grant_type: 'authorization_code';
```

Comment:
I think these two do not need to be specified here. One is a constant and the other is already specified in the SDK client instantiation. See how the SSO token endpoint fills these in:

https://github.com/workos/workos-node/blob/ddcd0ac0330e637a9b29b6bbfbf0c8e06b95e33b/src/sso/sso.ts#L83-L84

Reply:
ah, great point :+1:

Reply:
done.